spec/presenters/content_item_presenter_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe ContentItemPresenter do
+  let(:policy) {
+    PolicyArea.new(
+      name: "Tea and biscuits",
+      slug: "tea-and-biscuits",
+      description: "Tea and biscuits are popular among many people who live in the UK.",
+      content_id: "f61106bd-fb70-4003-a85a-71e5d52bea01",
+    )
+  }
+
+  let(:presenter) { ContentItemPresenter.new(policy) }
```

Comment:
Doesn't need changing, but you can do `described_class.new(policy)` for the same effect, which is quite nice.